.eslintrc.js

```diff
@@ -34,6 +29,7 @@ module.exports = {
     ],
     'no-console': 0,
     'no-bitwise': 0,
+    'max-classes-per-file': 0,
```

Comment:
0 disables the eslint rule, as source code is using more than 1 class `npm run lint` was failing without this option disabled.

Reply:
We could define the max number of classes per file if you prefer rather than totally disabling the rule.
https://eslint.org/docs/rules/max-classes-per-file

Reply:
I'm really against the dogmatic (Java) approach based around classes. Classes are tools like everything else and limiting them per file just limits the toolset. Lets use `0` and be done with that. :-)